core-primitives/test/src/mock/trusted_operation_pool_mock.rs

```diff
@@ -16,6 +16,15 @@
 
 */
 
+#[cfg(all(not(feature = "std"), feature = "sgx"))]
+use crate::sgx_reexport_prelude::*;
+
+#[cfg(feature = "sgx")]
+use std::sync::SgxRwLock as RwLock;
+
+#[cfg(feature = "std")]
+use std::sync::RwLock;
+
```

Comment:
The crate is now available in `std` and `sgx`, so I had to make these adjustments.